scripts/test

```diff
@@ -26,19 +35,31 @@ for DOCKER_VERSION in $(ls -1d */ | cut -d/ -f1 | grep -E $SEMVER_REGEX ); do
 			IMAGE=${TEST_OS_IMAGE_NAME[$index]}
 			for TAG in ${TEST_OS_IMAGE_TAG[$index]}; do
 				echo "==> Testing $DOCKER_VERSION.sh at ${IMAGE}:${TAG}..."
-				COMMAND="sh /install.sh"
+				COMMAND="bash /install.sh"
 				# Patch for oracle linux 8 docker image as yum is not installed
 				if [ "$IMAGE" == "oraclelinux" ] && [ "$TAG" == "8" ]; then
-					COMMAND="dnf install -y yum && sh /install.sh"
+					COMMAND="dnf install -y yum && bash /install.sh"
 				fi
 				docker run --rm -it --privileged \
-					-v $PWD/../dist/$DOCKER_VERSION.sh:/install.sh \
+					-v /install-docker/dist/$DOCKER_VERSION.sh:/install.sh \
```

Comment:
Is not better to use relative path within repo?? It should work in a dapper or not dapper env